packages/backend/src/App.ts

```diff
@@ -183,15 +184,15 @@ export default class App {
             App.initNodeProcessMonitor();
         }
 
-        const expressApp = this.initExpress();
+        const expressApp = await this.initExpress();
```

Comment:
Issuer discovery requires an async call at setup time, if configured to use `private_key_jwt` for Azure.